pkgs/by-name/pn/pnfft/package.nix

```diff
@@ -0,0 +1,59 @@
+{ autoconf
+, automake
+, fetchurl
+, fftwMpi
+, gsl
+, lib
+, libtool
+, llvmPackages
+, pfft
+, precision ? "double"
+, stdenv
+}:
+
+assert lib.elem precision [ "single" "double" "long-double" ];
+
+let
+  fftw' = fftwMpi.override { inherit precision; };
+  pfft' = pfft.override { inherit precision; };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pnfft-${precision}";
+  version = "1.0.7-alpha";
+
+  src = fetchurl {
+    url = "https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/software/pnfft-${finalAttrs.version}.tar.gz";
+    hash = "sha256-/aVY/1fuMRl1Q2O7bmc5M4aA0taGD+fcQgCdhVYr1no=";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    libtool
```

Comment:
does the `autoreconfHook` work here instead (and dropping the bootstrap script)?